readme.md

```diff
@@ -92,6 +92,33 @@ const fileType = require('file-type');
 })();
 ```
 
+Or from a _tokenizer_:
+
+A [_tokenizer_](https://github.com/Borewit/strtok3#tokenizer) propagates the internal read functions, 
+allowing alternative transport mechanisms, to access files, to be implemented and used.
+
+An example is [@tokenizer/http](https://github.com/Borewit/tokenizer-http) which requests data using range-requests.
+A difference with a conventional stream, is that it for example allows to only read the first 6, and the last 128 bytes.
+
+```js
+const {HttpTokenizer} = require('@tokenizer/http');
+const FileType = require('file-type');
+
+const config = {
+  avoidHeadRequests: true
+};
+
+const audioTrackUrl = 'https://test-audio.netlify.com/Various%20Artists%20-%202009%20-%20netBloc%20Vol%2024_%20tiuqottigeloot%20%5BMP3-V2%5D/01%20-%20Diablo%20Swing%20Orchestra%20-%20Heroines.mp3';
+
+(async () => {
+  const httpTokenizer = HttpTokenizer.fromUrl(audioTrackUrl, config);
+  await httpTokenizer.init();
+
+  const fileType = await FileType.fromTokenizer(httpTokenizer);
+  console.log(fileType);
+  // { ext: 'mp3', mime: 'audio/mpeg' }
```

Comment:
Use tab-indentation.

Reply:
Oh yes, I cannot find an option in IntelliJ to override tab indents for markdown.

Reply:
Fixed them one more time